### ISSUE TITLE
Problem in box plots 3-5 with new version of matplotlib

### ISSUE THREAD

**Issue by vassilisvas** [jbmouret/matplotlib_for_papers#4]
It seems that there is a change in the new version of matplotlib and boxplot3.py, boxplot4.py and boxplo5.py do not work anymore:

/usr/lib/python2.7/dist-packages/matplotlib/**init**.py:874: UserWarning: text.fontsize is deprecated and replaced with font.size; please use the latter.
  warnings.warn(self.msg_depr % (key, alt_key))
Traceback (most recent call last):
  File "boxplot3.py", line 48, in <module>
    bp['fliers'][i \* 2].set(markerfacecolor=colors[i],
IndexError: list index out of range

In boxplot3.py, the error can be addressed by making the following change. Replace lines 48-53 (https://github.com/jbmouret/matplotlib_for_papers/blob/master/src/boxplot3.py#L48) with just this:

bp['fliers'][i].set(markerfacecolor=colors[i],
                    marker='o', alpha=0.75, markersize=6,
                    markeredgecolor='none')

**Comment by costashatz** [comment]
There has been a change in recent `matplotlib` versions. We should have only 1 `fliers` field per box.

``` py
bp['fliers'][i].set(markerfacecolor=colors[i],
                        marker='o', alpha=0.75, markersize=6,
                        markeredgecolor='none')
```

**Comment by dogoepp** [comment]
Was the issue solved ?

**Comment by jbmouret** [comment]
fixed in #11
